README.md

```diff
@@ -186,6 +195,31 @@ A file to correct invalid or missing package metadata, and to set the concluded
 | ------ | ----- | ---------------- | ------------- |
 | YAML / JSON | Global | `$ORT_CONFIG_DIR/curations.yml` | Empty (n/a) |
 
+[How to fix text provider script](./docs/how-to-fix-text-provider-kts.md)
+
+A Kotlin script that enables the injection of how-to-fix texts in markdown format for ORT issues into the reports.
+
+| Format | Scope | Default location | Default value |
+| ------ | ----- | ---------------- | ------------- |
+| Kotlin script | Global | `$ORT_CONFIG_DIR/how-to-fix-text-provider.kts` | Empty (n/a) |
+
+[License configuration file](./docs/config-file-licenses-yml.md)
+
+A file that contains user-defined categorization of licenses.
+
+| Format | Scope | Default location | Default value |
+| ------ | ----- | ---------------- | ------------- |
+| YAML / JSON | Global | `$ORT_CONFIG_DIR/licenses.yml` | Empty (n/a) |
+
+[Package configuration file or directory](./docs/config-file-package-configuration-yml.md)
```

Comment:
That's already documented (see line 241).

Reply:
Removed.